app/javascript/packs/storiesList.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file Manages logic to add active class on items when it's a touch device
+ */
+const isTouchDevice = 'ontouchstart' in window || navigator.maxTouchPoints > 0;
+if (isTouchDevice) {
+  Array.from(document.getElementsByClassName('active')).forEach((el) =>
+    el.classList.remove('active'),
+  );
+
+  document.getElementById('main-content').addEventListener('click', (event) => {
+    const clickedEl = event.target;
+    if (
+      clickedEl.classList.contains('bm-initial') ||
+      clickedEl.classList.contains('bm-success')
+    ) {
+      //do nothing
+    } else if (clickedEl.parentNode.classList.contains('crayons-story')) {
+      clickedEl.parentNode.classList.add('active');
+    } else if (
+      clickedEl.parentNode.parentNode.classList.contains('crayons-story')
+    ) {
+      clickedEl.parentNode.parentNode.classList.add('active');
+    } else if (
+      clickedEl.parentNode.parentNode.parentNode.classList.contains(
+        'crayons-story',
+      )
+    ) {
+      clickedEl.parentNode.parentNode.parentNode.classList.add('active');
+    }
+  });
+}
```

Comment:
This is the section (unless confirmed otherwise) we could remove

Reply:
I tested this out on production and the code does get called and adds the `active` class when the elements with the `.crayons-selector` are clicked, but there appears to be no relavant styling in the project aside from this.

https://github.com/forem/forem/blob/4da6fc096cbb9c4ba6cc60cf6017942f614cf77c/app/assets/stylesheets/articles.scss#L292-L295

But that looks like that can be cleaned up as well. Just going to tag @aitchiss in case there was something I missed about this.

Reply:
I see a game of tag unfolding here 🤣

Reply:
I think we're all in agreement that this is no longer needed 🎉 

Agree the `.active` class itself can be removed too. That TODO comment cracked me up earlier - seems like we've been wondering about this for a while 😆